Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -57,24 +57,18 @@ bool fontNameIsSystemFont(CFStringRef fontName)
     return CFStringGetLength(fontName) > 0 && CFStringGetCharacterAtIndex(fontName, 0) == '.';
 }
 
-static RetainPtr<CFArrayRef> variationAxes(CTFontRef font, ShouldLocalizeAxisNames shouldLocalizeAxisNames)
+static RetainPtr<CFArrayRef> variationAxesWithNonLocalizedAxesNames(CTFontDescriptorRef fontDescriptor)
```

Comment:
Nit: The changes here have the effect of swapping the order of these functions. I understand that this first function is now only needed when `USE(KCTFONTVARIATIONAXESATTRIBUTE)` is true (so you moved it into the macro below), but it makes the diff a little harder to follow (to see before/after).

Reply:
oh yeah, it does make it harder to read, sorry. I've actually moved it because `variationAxesWithNonLocalizedAxesNames` is now used by `variationAxes`, so I wanted to avoid having to declare the static function before.

Reply:
Ah, that makes sense. No need to change the ordering.